Component/PetList/PetList.js

```diff
@@ -0,0 +1,20 @@
+import styled from "styled-components";
+import pets from "@/Lib/Data";
+import Pet from "../Pet/Pet";
+
+const StyledPetList = styled.div`
+  display: flex;
+  flex-direction: column;
+`;
+
+export default function PetList() {
+  return pets.length > 0 ? (
+    <StyledPetList>
+      {pets.map((pet, index) => {
+        return <Pet key={index} petData={pet} />;
```

Comment:
If your `StyledPetList` is going to be an `ul`, you need to add a `li` element here in your `map`.